MODULE.bazel

```diff
@@ -0,0 +1,51 @@
+module(
+    name = "io_bazel_rules_docker",
```

Comment:
Should we just name this `rules_docker`?

Reply:
Done.